_dashboards/admin-ui-index/datastream.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Data streams management
+parent: Index and snapshot management in OpenSearch Dashboards
+nav_order: 20
+---
+
+# Data streams management
+Introduced 2.6
+{: .label .label-purple }
+
+In OpenSearch Dashboards, the **Index Management** section allows you to view and manage [data streams]({{site.url}}{{site.baseurl}}/opensearch/data-streams/).
```

Comment:
@ariamarble Replace "section" with "application," the Dashboards team uses the term "application" to refer to the items in the home screen navigation pane.